h2o-py/h2o/explanation/_explain.py

```diff
@@ -2,15 +2,22 @@
 
 import random
 from contextlib import contextmanager
-from collections import OrderedDict, Counter
+from collections import OrderedDict, Counter, defaultdict
 
 import h2o
 import numpy as np
 import matplotlib
 import matplotlib.colors
 import matplotlib.figure
 from h2o.utils.ext_dependencies import get_matplotlib_pyplot
+from h2o.exceptions import H2OValueError
 
+try:
+    import matplotlib.pyplot as plt
+except ImportError:
+    # Possibly failed due to missing tkinter in old matplotlib in python 2.7
+    matplotlib.use("Agg")
+    import matplotlib.pyplot as plt
```

Comment:
I'm confused, I thought you wrote `get_matplotlib_pyplot` and used it on-demand to avoid potential conflicts/warnings when using this.

Reply:
Good catch, thank you!  I might have started this branch before the other one got merged I'll check and report here.

Reply:
🤦 I must have rebase without noticing this. Fixed now.